src/cli/commands/beacon.ts

```diff
@@ -15,17 +15,22 @@ import deepmerge from "deepmerge";
 import {getTomlConfig, IConfigFile} from "../../util/file";
 import defaults from "../../node/defaults";
 import {ILogger} from "../../logger";
+import {promptPassword} from "../../util/io";
 
 interface IBeaconCommandOptions {
   db: string;
   depositContract: string;
   eth1RpcUrl: string;
   rpc: string;
   configFile: string;
-  loggingLevel: string;
+  validator: boolean;
```

Comment:
better to group all validator params under same key if that what you mean

Reply:
Well this is the beacon command, why is it concerned with a validator?

Reply:
Thats how we're handling the bundled validator. It's bundled as an option in the beacon command.

Reply:
I think @mpetrunic  is saying we should have the option look like, eg:
```
validator?: {
  key: string;
  db?: string;
}
```

Reply:
For sure, but im curious as to why we need it?

Reply:
We need options as if we were running the validator as a standalone, because we're running the validator alongside the beacon chain when we're running in "bundled mode" by extending the beacon command.

Reply:
oh my... i didn't udnerstand the architecture at all...

I assumed `start` command would trigger a validator and beacon chain separately through their own respective cli commands (files?).

Reply:
The `start` command is for starting the bundled validator. The bundled validator needs some initialized property of the `BeaconNode` for starting as it doesn't use the network layer through `RpcClientOverWs`, instead uses the `RpcClientOverInstance`. That's why the `start` command doesn't trigger a validator and beacon chain separately.

Reply:
Than you would have to expose rpc over some server, which would be annoying for testing lodestar locally as yyou would have to configure each node to expose rpc on separate port and stuff

Reply:
> I think @mpetrunic is saying we should have the option look like, eg:
> 
> ```
> validator?: {
>   key: string;
>   db?: string;
> }
> ```

Updated `IBeaconCommandOptions` and `BeaconNodeCtx` according to this.